spec/predicates/vsa.md

```diff
@@ -1,11 +1,279 @@
-# Predicate type: SLSA Verification Summary (VSA)
+# Predicate type: Verification Summary Attestation (VSA)
 
-Type URI: https://slsa.dev/verification_summary/v1
+Type URI: https://in-toto.io/attestation/verification_summary/v1
 
-Version: 1.0
+Version: 1.1
```

Comment:
+1, I would expect the 1.0 SLSA VSA attestation to be deprecated in favor of a 0.1 in-toto VSA since there is a fair amount of incompatible changes between these two.

Reply:
Since the URI's domain is changing, can we drop the predicate version to v0.x?